csvw/csvw.go

```diff
@@ -357,3 +353,18 @@ func newColumn(title, label string) Column {
 
 	return c
 }
+
+func GenerateCustomTitle(dims []dataset.VersionDimension) string {
+	var title string
+	l := len(dims)
+	for i, d := range dims {
+		if i == 0 {
+			title += d.Label
+		} else if i == (l - 1) {
+			title += " and " + d.Label
+		} else {
+			title += ", " + d.Label
+		}
```

Comment:
Very minor but how would you feel about this little change which in my opinion improves readability?
```suggestion
		if i == (l - 1) {
			title += " and "
		} else if i > 0 {
			title += ", "
		}
		title += d.Label
```
We could also calculate the last index once before the loop and store it in a variable named accordingly for easier reading.